[2024-09-11T03:57:21.663+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T03:57:21.684+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2023-02-19T00:00:00+00:00 [queued]>
[2024-09-11T03:57:21.696+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2023-02-19T00:00:00+00:00 [queued]>
[2024-09-11T03:57:21.697+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T03:57:21.719+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2023-02-19 00:00:00+00:00
[2024-09-11T03:57:21.726+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=29281) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T03:57:21.727+0000] {standard_task_runner.py:63} INFO - Started process 29290 to run task
[2024-09-11T03:57:21.727+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgers_hooks_v04', 'postgres_to_s3', 'scheduled__2023-02-19T00:00:00+00:00', '--job-id', '4543', '--raw', '--subdir', 'DAGS_FOLDER/17_dag_with_postgress_hook.py', '--cfg-path', '/tmp/tmp810dlk4q']
[2024-09-11T03:57:21.729+0000] {standard_task_runner.py:91} INFO - Job 4543: Subtask postgres_to_s3
[2024-09-11T03:57:21.779+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2023-02-19T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T03:57:21.872+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgers_hooks_v04' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-02-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-19T00:00:00+00:00'
[2024-09-11T03:57:21.873+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T03:57:21.887+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:212 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-09-11T03:57:21.895+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T03:57:21.905+0000] {17_dag_with_postgress_hook.py:38} INFO - Saved orders data in text file: dags/get_orders_20230219.txt
[2024-09-11T03:57:21.913+0000] {base.py:84} INFO - Using connection ID 'minio_s3_conn' for task execution.
[2024-09-11T03:57:21.914+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2024-09-11T03:57:22.237+0000] {17_dag_with_postgress_hook.py:48} INFO - Orders file /tmp/tmp6f9b_yu_20230219 has been pushed to S3!
[2024-09-11T03:57:22.237+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-11T03:57:22.238+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T03:57:22.259+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgers_hooks_v04, task_id=postgres_to_s3, run_id=scheduled__2023-02-19T00:00:00+00:00, execution_date=20230219T000000, start_date=20240911T035721, end_date=20240911T035722
[2024-09-11T03:57:22.302+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T03:57:22.318+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T03:57:22.324+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
